daphne/src/lib.rs

```diff
@@ -469,6 +485,7 @@ pub enum DapHelperTransition<M: Debug> {
 #[serde(rename_all = "snake_case")]
 pub enum VdafConfig {
     Prio3(Prio3Config),
+    Prio2 { dimension: u32 },
```

Comment:
could you create a `Prio2Config` type, to hide the similar interface between prio versions..

Reply:
For Prio3, the Prio3Config expresses the "data type", e.g., count, sum, histogram, etc. Prio2 is not as flexible: It supports count vectors of any length, where length is `dimension`.